client/webserver/site/src/js/app.js

```diff
@@ -346,6 +354,21 @@ export default class Application {
     }
     return true
   }
+
+  /**
+   * signOut call to /api/logout, if response with no errors occured reload the page,
+   * otherwise will show a notification
+   */
+  async signOut () {
+    const res = await postJSON('/api/logout')
+    if (!this.checkResponse(res)) {
+      this.page.profileBox.style.display = 'none'
+      return
+    }
+    State.clearAllStore()
+    State.clearAllCookies()
```

Comment:
Clearing all cookies will delete user settings, which is currently just dark mode. Once the auth token is deleted server-side, there is no security-sensitive info in the cookies. If it makes everyone feel better, you can clear the auth token, but don't do a blanket delete of all cookies.